lib/mix/tasks/phoenix.new.ex

```diff
@@ -33,6 +34,12 @@ defmodule Mix.Tasks.Phoenix.New do
 
     copy_from template_dir, path, application_name, &EEx.eval_file(&1, binding)
     copy_from static_dir, Path.join(path, "priv/static"), application_name, &File.read!(&1)
+
+    # TODO decide to auto npm install or not
+    if npm_path && Mix.env == :dev do
```

Comment:
> Should we provide an option to disable npm install?

Probably. Otherwise, if you're just building an API and happen to have node installed, you need to wait for `npm install` to finish, then rm -rf node_modules.

> Should we provide an option to disable brunch?

I don't think we need it, because you can just set, `watchers: []` in your `dev.exs` to disable brunch from running. You will have to delete brunch-config and package.json though manually. wdyt?

> If NPM is not available, I think we should emit a message to the user: "You don't have npm installed. This means we were not able to install all packages... yada yada... install npm and run npm instal inside the app"

+1